pages/admin/index.tsx

```diff
@@ -37,15 +37,11 @@ import useDebounce from '@tools/hooks/useDebounce'; // Debounce hook
 import { Column } from 'react-table';
 
 // Map uppercase enum strings to lowercase
-const permitTypeString: Record<PermitType, string> = {
+const permitTypeString: Record<string, string> = {
```

Comment:
Just wondering why this was changed? I think it should be working fine

Reply:
Without using the `PermitTypeProps`, it interprets the result as an `any` and doesn't allow for reading from `permitTypeString` since it might not be a `PermitType`.
[Exhibit A](https://railway.app/project/afc837ea-aa06-4d86-8732-287740d12581/deployments?id=3b3e73a7-e9cc-4828-8ee5-e2d600b5f5c0)
[Exhibit B](https://railway.app/project/afc837ea-aa06-4d86-8732-287740d12581/deployments?id=946771c2-4188-4a4c-97bf-a6cd156f95e0)
[Exhibit C](https://railway.app/project/afc837ea-aa06-4d86-8732-287740d12581/deployments?id=1f4921c8-40d4-4a76-a3dd-988014a4469c)

If you have a better fix then lmk and we can address it separately

Reply:
Got it. Will add to backlog.